runtime/centrifuge/src/xcm.rs

```diff
@@ -147,26 +156,26 @@ pub type Barrier = (
 );
 
 /// Means for transacting the fungibles assets of this parachain.
+/// todo(nuno): consider using `MultiCurrencyAdapter` instead
```

Comment:
Should this be fixed before merging?

Reply:
Thanks for the catch. Nop, I left this for me to have a look at the `MultiCurrencyAdapter` that Acala uses but leaving things as are.